content/posts/packit-041.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Packit 0.4.0 & 0.4.1"
+date: 2019-05-18
+draft: false
+---
+
+It's been over a month since 0.3.0 and newly released 0.4.0 (and patch release 0.4.1) again bring a lot of new features and improvements.
```

Comment:
```suggestion
It's been over a month since we released packit "0.3.0". Here comes packit 0.4.0 (and patch release 0.4.1) and as always they bring a lot of new features and improvements.
```